src/css/modules/pages/home.css

```diff
@@ -1,18 +1,19 @@
 .hero--home .hero__title {
   --max-size: var(--font-size-9);
-  text-align: center;
 }
 
 .hero--home .hero__body {
   font-size: var(--font-size-6);
-  margin-left: auto;
-  margin-right: auto;
-  text-align: center;
 }
 
-.home-model .content-block__intro,
-.home-portfolio .content-block__intro {
-  margin-left: auto;
+/* Should I be using em or px? */
+@media (max-width: 990px) {
```

Comment:
We likely want a `min-width` here and for most media queries. That way we're saying that small screens are the baseline styles and only change styles if the screen reaches a certain width. That's a mobile-first approach that I try to follow. Effectively they do the same thing, but just a shift in thinking to where we're layering on style and functionality as more screen is available

Reply:
Mobile-first and onward, got it 👍.